diff --git a/tools/SGXPlatformRegistration/package/installer/common/sgx-ra-service/Makefile b/tools/SGXPlatformRegistration/package/installer/common/sgx-ra-service/Makefile
index 1f3efdb..42d7ed6 100644
--- a/tools/SGXPlatformRegistration/package/installer/common/sgx-ra-service/Makefile
+++ b/tools/SGXPlatformRegistration/package/installer/common/sgx-ra-service/Makefile
@@ -47,7 +47,7 @@ RAD_CONF_NAME=mpa_registration_tool.service
 RAD_CONF_DEL=mpa_registration.conf
 RAD_CONF_PATH=/lib/systemd/system
 else
-$(error "Unsupported platform - neither systemctl nor initctl is found!")
+@exit 0
 endif
 endif