diff --git a/linux/installer/common/psw/Makefile b/linux/installer/common/psw/Makefile
index e46cf53..5b74add 100644
--- a/linux/installer/common/psw/Makefile
+++ b/linux/installer/common/psw/Makefile
@@ -50,11 +50,11 @@ install:
 	install -d $(USR_LIB_PATH)
 	cp -r $(SOURCE_FOLDER)/* $(SOURCE_INSTALL_PATH)
 	install $(SCRIPTS_FOLDER)/* $(SCRIPTS_INSTALL_PATH)
-	mv $(PSW_LIB_PATH)/libsgx_uae_service.so $(USR_LIB_PATH)
-	mv $(PSW_LIB_PATH)/libsgx_urts.so $(USR_LIB_PATH)
-	mv $(PSW_LIB_PATH)/libsgx_enclave_common.so $(USR_LIB_PATH)
+	sudo mv $(PSW_LIB_PATH)/libsgx_uae_service.so $(USR_LIB_PATH)
+	sudo mv $(PSW_LIB_PATH)/libsgx_urts.so $(USR_LIB_PATH)
+	sudo mv $(PSW_LIB_PATH)/libsgx_enclave_common.so $(USR_LIB_PATH)
 	rmdir $(PSW_LIB_PATH)
 	cd $(USR_LIB_PATH) && \
-	mv libsgx_enclave_common.so libsgx_enclave_common.so.$(ECL_VER) && \
-	ln -fs libsgx_enclave_common.so.$(ECL_VER) libsgx_enclave_common.so.$(shell echo $(ECL_VER) |awk -F. '{print $$1}') && \
-	ln -fs libsgx_enclave_common.so.$(shell echo $(ECL_VER) |awk -F. '{print $$1}') libsgx_enclave_common.so
\ No newline at end of file
+	sudo mv libsgx_enclave_common.so libsgx_enclave_common.so.$(ECL_VER) && \
+	sudo ln -fs libsgx_enclave_common.so.$(ECL_VER) libsgx_enclave_common.so.$(shell echo $(ECL_VER) |awk -F. '{print $$1}') && \
+	sudo ln -fs libsgx_enclave_common.so.$(shell echo $(ECL_VER) |awk -F. '{print $$1}') libsgx_enclave_common.so
\ No newline at end of file
diff --git a/linux/installer/common/psw/install.sh b/linux/installer/common/psw/install.sh
index 59f3ed6..8df19e2 100755
--- a/linux/installer/common/psw/install.sh
+++ b/linux/installer/common/psw/install.sh
@@ -42,7 +42,7 @@ AESM_PATH=$PSW_DST_PATH/aesm
 # Install the AESM service
 
 id -u aesmd &> /dev/null || \
-    /usr/sbin/useradd -r -U -c "User for aesmd" \
+    sudo /usr/sbin/useradd -r -U -c "User for aesmd" \
     -d /var/opt/aesmd -s /sbin/nologin aesmd
 
 mkdir -p /var/opt/aesmd
